configure.ac

```diff
@@ -2947,8 +2947,8 @@ dnl stream in big-endian byte-order
 AC_CHECK_HEADERS([uuid.h], [
   AC_CHECK_FUNCS([uuid_create uuid_enc_be], [
     have_uuid=yes
-    LIBUUID_CFLAGS=
-    LIBUUID_LIBS=
+    LIBUUID_CFLAGS=${LIBUUID_CFLAGS:-""}
+    LIBUUID_LIBS=${LIBUUID_LIBS:-""}
```

Comment:
I think we should use `${VAR-default}` throughout the PR. We only want to override if the variable is _not_ set, right?

```sh
$ TEST=
$ echo ${TEST-default}

$ echo ${TEST:-default}
default
$ unset TEST
$ echo ${TEST-default} 
default
$ echo ${TEST:-default}
default
```

Reply:
good point!